src/lib.rs

```diff
@@ -366,6 +313,13 @@ pub fn is_key_down(key_code: KeyCode) -> bool {
     context.keys_down.contains(&key_code)
 }
 
+/// The characters typed this frame.
+///
+/// Clears after every call to next_frame().await
```

Comment:
It looks like its not quite true - it clears after each `typed_characters` call, isnt it? 

Maybe return &[char] instead and either avoid one allocation and do get the same output on each `typed_character` during the frame?

Reply:
Whoops, yes, I had this left over from an earlier implementation which provided all of the characters typed since the last call to `typed_characters`, which I removed because I was afraid that people would who never called `typed_characters` would end up with these huge strings allocated which they would never use.

I agree, but I think it makes more sense to return `&str` than `&[char]`.